LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/track-sizing/masonry-track-sizing-overflow-right-side.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+    
+<head>
+    <meta charset="utf-8">
+    <title>CSS Grid Test: Verify correct behavior of track-sizing</title>
+    <link rel="author" title="Brandon Stewart" href="mailto:brandonstewart@apple.com">
+    <link rel="help" href="https://drafts.csswg.org/css-grid-3/#track-sizing">
+    <link rel="match" href="masonry-track-sizing.html">
+</head>
+
+<body>
+<style>
+grid {
+  display: grid;
+  grid-template-columns: 1fr auto;
+  grid-template-rows: masonry;
+  width: 300px;
+  height: 100px;
+}
+
+box1 {
+    width: 1fr;
+    height: 50px;
+    background-color: red;
+}
+
+box2 {
+    height: 50px;
+    width: 50px;
+    background-color: blue;
+}
+
+box3 {
+    width: 1fr;
```

Comment:
The 'fr' unit is a special thing that can only be used in the template of grid tracks, it's not meaningful to use it to size other boxes.

Reply:
Thanks. Deleted.